cms/articles/models.py

```diff
@@ -232,3 +232,8 @@ def is_latest(self) -> bool:
             .first()
         )
         return bool(self.pk == latest_id)  # to placate mypy
+
+    @property
+    def search_index_content_type(self) -> str:
+        """Returns the content type string for the search index."""
+        return "bulletin"
```

Comment:
question: why not an actual property on the class?

Reply:
Good question, now thinking about it we can just have a static, class-level value (rather than a dynamic, instance-level property that can be overridden or extended as we don't need that from my understanding).

I have updated this on the other models too now.